bridge/test/Federation_test.js

```diff
@@ -14,6 +14,8 @@ contract('Federation', async function (accounts) {
     const federator2 = accounts[3];
     const federator3 = accounts[4];
     const bridge = utils.getRandomAddress();
+    const TOKEN_TYPE_COIN = 0;
```

Comment:
maybe I'd make some fake enum [like they do here](https://www.sohamkamani.com/javascript/enums/#enumerations-with-objects), and just use the `COIN` member in the tests below (and I'd remove the commented constant)